README.md

```diff
@@ -41,6 +41,11 @@ The message to emit when the check fails. All other inputs can be specified in
 the message using `${}` syntax, e.g. `${file-pattern}` for the `file-pattern`
 input. All values will be quoted for easy identification of any whitespace.
 
+### `token`
+
+Github token to use for private repositories. Falls back to anonymous access if
```

Comment:
```suggestion
A GitHub auth token to use for private repositories. Falls back to anonymous access if
```

Is there a good URL to point users to for what sort of token to request?

Reply:
In most cases probably using [`${{ secrets.GITHUB_TOKEN }}`](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) should be enough.